media/asciidoctor-editor.css

```diff
@@ -200,13 +200,12 @@ body {
 }
 
 body {
-  /* DISABLED background: #fff; */
-  /* DISABLED color: rgba(0, 0, 0, .8); */
-  padding: 0;
+  font-family: var(--asciidoc-font-family, -apple-system, BlinkMacSystemFont, "Segoe WPC", "Segoe UI", system-ui, "Ubuntu", "Droid Sans", sans-serif);
+  font-size: var(--asciidoc-font-size, 14px);
+  padding: 0 26px;
+  line-height: var(--asciidoc-line-height, 22px);
+  word-wrap: break-word;
   margin: 0;
-  /* DISABLED font-family: "Noto Serif", "DejaVu Serif", serif; */
-  /* DISABLED TODO: decide whether to use serif font
-  font-family: serif; */
   font-weight: 400;
   font-style: normal;
   line-height: 1;
```

Comment:
Thank you :tada: 

Font family and font size are working nicely.

Line-height is more tricky.

On inspection, I think line 211, `line-height: 1` may be overriding line 206 `line-height: var(--asciidoc-line-height, 22px);`. After removing this line I am still not entirely successful.

Sometimes this works. I am simply putting values in the preview setting on one side and waiting for the refresh on the other side. I am trying: [0.9, 1.1, 1.2, 1.5, 2, 3, 4, 5], all of which should be effective as I understand it (we are specific a multiple of the font size).

https://developer.mozilla.org/en-US/docs/Web/CSS/line-height

Looking at it more I think it may be related to many elements having overrides on line-height in the stylesheet and the examples that I'm working with.

For example:

```
p {
  font-family: inherit;
  font-weight: 400;
  font-size: 1em;
  line-height: 1.6;
  margin-bottom: 1.25em;
  text-rendering: optimizeLegibility
}
```

As I understand it, line-height is inherited but on the computed value which I  think makes it difficult to cascade a single line-height setting through a whole document.

https://tympanus.net/codrops/css_reference/line-height/

I don't think this is worth a great deal of effort...

Reply:
Nice catch, you are absolutely right.
I think the line-height should first and foremost apply on paragraphs. The Mardown extension is using a fixed value for `h1` which I think is a good, since you probably don't want to use the same value for paragraphs and headings.